tools/lorisform_parser.php

```diff
@@ -151,7 +152,10 @@ function parseElements($elements, $groupLabel="")
 
         case "date":
             $options = "{@}";
-            if (array_key_exists('options', $element)) {
+            if (array_key_exists('options', $element)
+                && isset($el['options']['minYear'])
+                && isset($el['options']['maxYear'])
+            ) {
```

Comment:
note: this behaviour is going to be different than current behaviour if only one of min/max is set, but I don't think that's ever the case and if it is it's probably a bug anyways, so I'm still going to merge this..